component-test/test_dbus_proxy.py

```diff
@@ -557,6 +557,52 @@ def test_methods(self, session_bus, service_on_outside, dbus_proxy, config):
         captured_stdout = dbus_send_process.communicate()[0]
         assert "My unique key" in captured_stdout
 
+    CONF_AVAILABLE_METHODS = """
+    {
+        "dbus-gateway-config-session": [{
+            "direction": "*",
+            "interface": "*",
+            "object-path": "*",
+            "method": ["Method1", "Method2"]
+        }],
+        "dbus-gateway-config-system": []
+    }
+    """
+
+    @pytest.mark.parametrize("config", [CONF_AVAILABLE_METHODS])
+    def test_all_available_methods(self, session_bus, service_on_outside, dbus_proxy, config):
+        """ This method is stands for testing process of method argument in configuration
+            to be ensure whether code can filter the method on following occasions :
+            * when one exact method name is provided (i.e CONF_SINGLE_METHOD)
+            * when wildcard character is used (i.e. CONF_ALL_METHODS)
+            * when an array of methods is provided (i.e CONF_FOUR_METHODS)
+        """
```

Comment:
The comment makes me think that we also should have more configs in the parametrization?